vulcanai/model_tests.py

```diff
@@ -185,20 +187,31 @@ def run_test(network, test_x, test_y, figure_path='figures', plot=True):
     if not plot:
         plt.close(fig.number)
     print ('Average AUC: : {:.4f}'.format(np.average(all_class_auc)))
-    return {
-        'accuracy': accuracy,
-        'macro_sensitivity': sens_macro,
-        'macro_specificity': spec_macro,
-        'avg_dice': np.average(dice),
-        'macro_ppv': ppv_macro,
-        'macro_npv': npv_macro,
-        'macro_f1': f1_macro,
-        'macro_auc': np.average(all_class_auc)
-    }
+
+    sens_by_class = {}
+    spec_by_class = {}
+    ppv_by_class = {}
+    npv_by_class = {}
+    f1_by_class = {}
+    for j in range(len(sens)):
+        sens_by_class["sens_class_" + str(j)] = sens[j]
```

Comment:
Because the df will now have a column for every metric in every class. So [sens_class_0, sens_class_1, ...., ppv_class_0, ppv_class_1,......]. Does that make sense?

Reply:
I feel like we can just keep it as a list. this just increases the complexity of the storage unit.